### PR TITLE
log-parser: remove kata-log-parser

### PR DESCRIPTION
**Pull request by snir911**
as it moves to kata-containers repo
Also remove it from teardown.sh as the log-parser isn't validating log
correctness well currently and the emitted errors are practically and
consistently ignored in most jobs

**NOTE:** adding do-not-merge label until it will be added to kata-containers

Related: github.com/kata-containers/kata-containers#4100
Depends-on: github.com/kata-containers/kata-containers#4165
Fixes: #4742
Signed-off-by: Snir Sheriber <ssheribe@redhat.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
Thanks for raising @snir911 - I'm a bit confused by the diffstat though. I'd expect to see the log parser files being deleted, and maybe some vendor files being deleted, but the stat shows other changes?!?

**Comment by snir911** [comment]
@jodh-intel i explained the teardown changes in the commit msg, it seem vendored code is out of sync, I'm trying to fix but something is seems to be messed before removing the log parser

**Review by jodh-intel — APPROVED** [review]
Thanks @snir911 - that looks better.

File: Gopkg.toml [hunk 1]
```diff
@@ -30,14 +30,6 @@ ignored = ["github.com/kata-containers/tests/kata-webhook"]
   name = "github.com/urfave/cli"
   version = "1.20.0"
 
-[[constraint]]
```
Comment (wainersm) [inline Gopkg.toml:33]: Hi @snir911 , asking just for curiosity: those constraints are used on log-parser so they are no longer used or they are leftovers of another clean ups?

**Review by wainersm — APPROVED** [review]
LGTM. Thanks @snir911

**Review by amshinde — APPROVED** [review]

File: Gopkg.toml (hunk 1 above)
Comment (snir911) [reply Gopkg.toml:33]: leftovers

**Pull request merged**